panda/src/audio/audioSound.h

```diff
@@ -174,6 +182,8 @@ class EXPCL_PANDA_AUDIO AudioSound : public TypedReferenceCount {
   }
   virtual TypeHandle force_init_type() {init_type(); return get_class_type();}
 
+  virtual std::vector<std::string> get_raw_comment() const;
```

Comment:
Put this further up, don't put this in the middle of the standard type boilerplate